packages/cli/scaffold/resource.template.js

```diff
@@ -1,86 +1,88 @@
-// get a single <%= LOWER_NOUN %>
-const get<%= CAMEL %> = (z, bundle) => {
-  const responsePromise = z.request({
-    url: `https://jsonplaceholder.typicode.com/posts/${bundle.inputData.id}`,
-  });
-  return responsePromise
-    .then(response => z.JSON.parse(response.content));
-};
-
 // get a list of <%= LOWER_NOUN %>s
-const list<%= CAMEL %>s = (z) => {
-  const responsePromise = z.request({
+const performList = (z, bundle) => {
```

Comment:
this is a lot of repeated code, but I think it's fine for now. we'll have to do more passes on these later anyway when we condense everything